auth/email_action_links_test.go

```diff
@@ -177,6 +179,27 @@ func TestPasswordResetLinkWithSettings(t *testing.T) {
 	}
 }
 
+func TestPasswordResetLinkWithSettingsNonExistingUser(t *testing.T) {
+	resp := `{
+		"error": {
+			"message": "EMAIL_NOT_FOUND"
+		}
+	}`
+	s := echoServer([]byte(resp), t)
+	defer s.Close()
+	s.Status = http.StatusBadRequest
+
+	link, err := s.Client.PasswordResetLinkWithSettings(context.Background(), testEmail, testActionCodeSettings)
+	if link != "" || err == nil {
+		t.Errorf("PasswordResetLinkWithSettings() = (%q, %v); want = (%q, error)", link, err, "")
+	}
+
+	want := "no user record found for the given email"
```

Comment:
Is this the error string shown in an error message?

In the other langs we made these messages "grammatical" with caps and punctuation, like "No user record found for the given email."

Reply:
This is the same error as below. Gonna mark it as duplicate.

Reply:
This is again consistent with Go conventions: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Thanks for confirming.